### PR TITLE
feature/SCAN-337-transition-to-new-image

### PR DESCRIPTION
**Pull request by a-mikhailova**

### CODE REVIEW

File: src/utils/convertAttributesToView.ts [hunk 1]
```diff
@@ -16,7 +16,7 @@ export interface AttributeView {
 }
 
 const getValueOrDefaultValue = (obj: LocalizedStringOrBoxedNumberWithDefault, lang = 'en'): string => {
-  return (obj as LocalizedStringWithDefault)[lang] ?? obj._.toString();
```
Comment (kpozdnikin) [inline src/utils/convertAttributesToView.ts:19]: Please add undefined check instead of removing toString

File: src/components/TokenCard.tsx [hunk 2]
```diff
@@ -32,7 +32,7 @@ const TokenCard: FC<TokenCardProps> = ({
     }
   }, []);
 
-  const imageUrl = useCheckImageExists(getImageURL(imagePath));
+  const imageUrl = useCheckImageExists(image.fullUrl || image.url);
```
Comment (kpozdnikin) [inline src/components/TokenCard.tsx:35]: Are you sure that old urls will come in pure format, not in json? getImageURL handles the 'json' case

File: src/api/graphQL/tokens/types.ts [hunk 3]
```diff
@@ -12,11 +12,9 @@ export interface Token {
...
   collection_id: number
   collection_name: string
-  data: {
-    image: {
-      url?: string;
-      fullUrl?: string;
-    }
+  image: {
+    url?: string;
+    fullUrl: string | null;
```
Comment (sswebcoder) [inline src/api/graphQL/tokens/types.ts:17]: Lost ipfsCid and urlInfix.

But maybe it not needed for frontend.

File: src/components/TokenCard.tsx (hunk 2 above)
Comment (sswebcoder) [reply src/components/TokenCard.tsx:35]: Yep, It's correct.
You can check this https://scan-api.demo.uniquenetwork.dev/v1/graphql


```GQL
query getTokens {
  tokens(where: { collection_id: { _eq: 1987 }, token_id: { _eq: 17 } }) {
    data {
      owner
      token_id
      token_name
      image
    }
  }
}
```

File: src/api/graphQL/tokens/types.ts (hunk 3 above)
Comment (a-mikhailova) [reply src/api/graphQL/tokens/types.ts:17]: I understood that if we are a scan that just displays the info, then fullUrl is enough for us. the other fields are needed to encode the token.  Do we have a need at the front to encode the token?

**Review by kpozdnikin — APPROVED** [review]

**Pull request merged**